### PR TITLE
Polished the tf output to be a closer match to Smartest export

### PR DESCRIPTION
**Pull request by rlaj**
Added various sorting helper methods to mimic how Smartest exports the tf file on 'Save'.  Tested with several different test flows.  However, I'm certain there are still some corner cases not covered that will generate different spacing and possible ordering, but this PR should clean up most diff issues.

### CODE REVIEW

**Comment by rlaj** [comment]
I assumed @ginty's changes would be transparent to these updates since they were in the lower-level ATP abstraction.  The tf rendering doesn't happen until later after all of the test flow logic has been built up and transformed into V93K specifics.  But @ginty may have other thoughts?

**Comment by ginty** [comment]
No problem to merge this once it's passing from my perspective, agree that there is no conflict with my work.

File: approved/v93k/custom_tests.tf [hunk 1]
```diff
@@ -1,81 +1,91 @@
...
-implicit_declarations
 
-end
------------------------------------------------------------------
 testmethodparameters
+
 tm_1:
-  "testerState" = "CONNECTED";
-  "testName" = "Functional";
+  "BadPractice" = "NO";
```
Comment (redxeth) [inline approved/v93k/custom_tests.tf]: What is this here?

Comment (rlaj) [reply approved/v93k/custom_tests.tf]: I think it's correct.  It's just re-ordering the parameters alphabetically, but the diff makes it look weird.
